Source Code/Ticket Management System/Ticket Management System.vcxproj

```diff
@@ -22,7 +22,7 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{BCF3B1AC-70F1-40FD-8EF0-6730B6E2D4E7}</ProjectGuid>
     <RootNamespace>TicketManagementSystem</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
```

Comment:
Can we please keep it at `10.0.17763.0` to not interfere with other peoples settings?